plugins/rake-fast/rake-fast.plugin.zsh

```diff
@@ -8,7 +8,11 @@ _rake_refresh () {
 }
 
 _rake_does_task_list_need_generating () {
-  [[ ! -f .rake_tasks ]] || [[ Rakefile -nt .rake_tasks ]]
+  [[ ! -f .rake_tasks ]] || [[ Rakefile -nt .rake_tasks ]] || (_is_rails_app && [[ lib/tasks -nt .rake_tasks ]])
```

Comment:
Yes, it seems rather strange for me too 😃  When I thought about how I could check whether new file has been created `lib/tasks` or existing file has been modified first thing I thought about was some kind of recursion check.

But this check seems to be working alright. I've tested it.
`stat -f%m lib/tasks` is updated both by adding new file and modifying existing one.

There is one thing I should clarify. When I do `git checkout some-branch` `stat -f%m lib/tasks` is changing. So than condition is resolved to `true`. I think it is how it should be. But I'd like you to confirm it.